src/Features/Missions/missionsSlice.js

```diff
@@ -0,0 +1,40 @@
+import { createSlice, createAsyncThunk } from '@reduxjs/toolkit';
+
+export const getMissionsAsync = createAsyncThunk(
+  'missions/getMissionsAsync',
+  async () => {
+    const response = await fetch('https://api.spacexdata.com/v3/missions');
+    if (response.ok) {
+      const data = await response.json();
+      const missions = data;
+      return missions;
+    }
+    return null;
+  },
+);
+
+const missionsSlice = createSlice({
+  name: 'missions',
+  initialState: {
+    missions: [],
+    joinMission: {},
+  },
```

Comment:
- Great to create initialState objects but I would highly recommend you to separate it from the reducers created above 🆗 [OPTIONAL]
- Try to add the isLoading key in **initialState** to measure if Fetch is working then be used in extra reducer as **pending** [OPTIONAL] 👏